src/V3Width.cpp

```diff
@@ -2792,20 +2792,65 @@ class WidthVisitor final : public VNVisitor {
     }
     void visit(AstMemberSel* nodep) override {
         UINFO(5, "   MEMBERSEL " << nodep << endl);
+        if (nodep->didWidth()) return;
```

Comment:
Maybe I missed it but not sure where didWidth is set in this.  Be careful to only do it after FINAL stage.

Reply:
Added to code paths that leave the MemberSel a MemberSel.